src/WebJobs.Extensions.DurableTask/EntityScheduler/EntityId.cs

```diff
@@ -44,7 +44,7 @@ public EntityId(string entityName, string entityKey)
 
         internal static string GetSchedulerIdFromEntityId(EntityId entityId)
         {
-            return $"@{entityId.EntityName}@{entityId.EntityKey}";
+            return $"@{entityId.EntityName}@{entityId.EntityKey}".ToLower();
```

Comment:
I believe we need to use `ToLowerInvariant()` instead of `ToLower()`. More discussion on why here: https://stackoverflow.com/questions/6225808/string-tolower-and-string-tolowerinvariant.